webhook/webhook.go

```diff
@@ -52,9 +52,9 @@ var (
 
 // ControllerOptions contains the configuration for the webhook
 type ControllerOptions struct {
-	// WebhookName is the name of the webhook we create to handle
+	// ResourceMutationWebhookName is the name of the webhook we create to handle
```

Comment:
This are normally called "Mutating Admision Webhooks", so `MutatingWebhookName`?

https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/

Reply:
I added `Resource` to qualify it against mutating webhooks that we may add at some other point. `ResourceMutatingWebhookName`?